wcomponents-theme/src/test/intern/resources/domTest.html

```diff
@@ -51,4 +51,9 @@
 			<legend id="leg2">nested legend</legend>
 		</fieldset>
 	</fieldset>
-</form>
\ No newline at end of file
+</form>
+<div>
+	<label id="aria-label">Name</label>
+	<input id="aria-text" type="text" aria-labelledby="aria-label aria-label2"/>
+	<label id="aria-label2">Name 2</label>
```

Comment:
Form bound elements (e.g. input) _should_ (not must) be bound by a form. If you have a reason for having this last input element outside of the form element please comment it so no one else tries to clean it up.